src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();

```

Comment:
better to create storage private and add some methods to interact with it

Reply:
I think this is a completely different logic for performing this task, I'm not sure, but it seems to me that the implementation is using a HashMap